translate/storage/test_mozilla_lang.py

```diff
@@ -96,3 +98,22 @@ def test_template(self):
         assert store2.units[0].target == ""
         assert store2.units[1].source == "Source2"
         assert store2.units[1].target == ""
+
+    @pytest.mark.parametrize(
+        "ok, target, istranslated", [
+            ("", "", False),  # Untranslated, no {ok}
+            (" ", "Source ", True),  # Excess whitespace, translated
+            (" {ok}", "Source", True),  # Valid {ok}
+            (" {ok} ", "Source", True),  # {ok} trailing WS
+            ("{ok}", "Source", True),  # {ok} no WS
+        ]
+                            )
```

Comment:
This raises some pep8 error: E126 continuation line over-indented for hanging indent